rules.md

```diff
@@ -58,34 +60,26 @@ Each **Idea** card may have a list of requirements that must be met before they
 
 ### **Event** 
 
-**Events** must be acted upon immediately when drawn and may be permanent or one-off and only affects the current player. Events can either be positive or negative, for example limiting a players ability to perform a specific action.
-
-### **Impact**
-
-When an **Insight** is **Disseminated**, the current player must draw a card from the top of the **Impact** deck, read the text aloud and then act on the text of the card imediately.
+**Events** must be acted upon immediately when drawn and may be permanent or one-off and only affects the current player. Events can either be positive or negative, for example limiting a players ability to perform a specific action. **Global Events** affect all Players.
```

Comment:
Are `Global Events` the new name for `Impact`s and get shuffled into the ideas deck, or are we not using the `Impact` cards now?

Reply:
Yes the Global Events are the old Impacts